Gemfile

```diff
@@ -4,3 +4,4 @@ source "https://rubygems.org"
 gemspec
 
 gem 'rails-controller-testing', group: :test
+gem 'shopify_browser_sniffer', github: 'Shopify/shopify_browser_sniffer', ref: 'shopify-matchers-and-monkey-patch'
```

Comment:
Waiting on this PR to ship so we can release the gem and make it available to `shopify_app`: https://github.com/Shopify/shopify_browser_sniffer/pull/2/files